src/aspire/basis/pswf_2d.py

```diff
@@ -50,18 +60,24 @@ def _build(self):
         # generate 1D indices for basis functions
         # self._indices = self.indices()
 
-        # precompute the basis functions in 2D grids
-        # self._precomp = self.precomp()
-        self.precomp()
-
-        # get normalized factors
-        # self._norms = self.norms()
+    def generate_grid(self):
+        if self.even:
```

Comment:
Again, isn't this handled by the `grid_1d` function in `utils.coor_trans`?

Reply:
have been removed in a new version.